arm64 support?

Hi y'all! Onboarding to Railway with the nextjs-prisma starter project.
Having an issue installing the CLI on an M1 (Apple Silicon) laptop:

```
% npm i -g @railway/cli
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! code 1
npm ERR! path /opt/homebrew/lib/node_modules/@railway/cli
npm ERR! command failed
npm ERR! command sh -c go-npm install
npm ERR! Installation is not supported for this architecture: arm64
npm ERR! Invalid inputs

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/croaky/.npm/_logs/2021-05-02T23_46_07_672Z-debug.log
```

Same for the `install.sh` script:

```
% sh -c "$(curl -sSL https://raw.githubusercontent.com/railwayapp/cli/master/install.sh)"
Error: arm64 machine type is not supported
Please create an issue so we can add support. https://github.com/railwayapp/cli/issues/new
```

Let me know how I can help test / debug this!

--- a/install.sh
+++ b/install.sh
@@ -93,7 +93,7 @@ get_machine() {
       echo "amd64" ;;
     "i386"|"i86pc"|"x86"|"i686")
       echo "i386" ;;
-    "armv6l"|"aarch64")
+    "arm64"|"armv6l"|"aarch64")
       echo "arm64"
   esac
 }

